torch/csrc/jit/mobile/interpreter.cpp

```diff
@@ -57,6 +57,11 @@ bool InterpreterState::run(Stack& stack) {
       auto inst_with_handle = code_->instructions_with_handles_.at(pc);
       Instruction inst = inst_with_handle.instruction;
       DebugHandle debug_handle = inst_with_handle.debug_handle;
+      // If no valid debug handle found then just log pc.
+      // This is possible when we did not save debug handles
+      // In that case post processing callback in EdgeProfile will fail
+      // to retrive usefule information.
```

Comment:
```suggestion
      // to retrieve useful information.
```

Reply:
Funny! I am compensating for lost e elsewhere. lol.